soc/v2/broadcom/bcm_vk/Kconfig

```diff
@@ -7,10 +7,11 @@ config SOC_FAMILY_BCMVK
 	bool
```

Comment:
cannot place comment on line above, but should this setting:
https://github.com/zephyrproject-rtos/zephyr/blob/fbdb44e26fe5febce19818c8584a863aeb5b9db5/soc/v2/broadcom/bcm_vk/Kconfig#L6-L7

be moved here:
https://github.com/zephyrproject-rtos/zephyr/blob/fbdb44e26fe5febce19818c8584a863aeb5b9db5/soc/v2/broadcom/bcm_vk/Kconfig.soc#L5

as it is referenced here (and other places as well):
https://github.com/zephyrproject-rtos/zephyr/blob/fbdb44e26fe5febce19818c8584a863aeb5b9db5/soc/v2/broadcom/bcm_vk/viper/Kconfig.soc#L6

Reply:
Fixed